### PR TITLE
Better User Guide section in README

### PR DESCRIPTION
**Pull request by KraXen72**
I rewrote the User Guide section to be more aligned with the new features of thumb-key 2.0.0. I have not tested if the in-app link to the guide still works, but i haven't changed the section's name, so it should.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,28 +50,40 @@ This project is a follow-up to the now unmaintained (and closed-source) [Message
...
-- Slide right on the `#` key to switch keyboard position.
-- Practice typing, and check your words per minute, using [monkeytype.com](https://monkeytype.com)
+- **Tap**, or **swipe** to type a letter.
+- **Swipe up** or **down** on `A` to capitalize. (If you changed the layout, the button next to `#`)  
+  <img width=60px height=60px src="https://i.postimg.cc/Znt2Ft9G/thumbkey-1-1.png">  
+- **Double tap** the space bar to type a comma, triple tap to type a period. More taps have more punctuation.
+- **Swipe left** on the backspace key to delete whole words.
+- **Swipe left** or **right** on the spacebar to move the cursor by 1 character.
+
+### Emoji Key
```
Comment (dessalines) [inline README.md:60]: You need a newline here, after every section header. Better yet run `prettier -w README.md` after you've made some changes. 

I'll add a lint for this in the CI to double check.

**Review by dessalines — COMMENTED** [review]
Looks much better, thx! Just run prettier on this file to fix any formatting issues.

**Comment by KraXen72** [comment]
I implemented the requested changes. Please take a look @dessalines

**Comment by KraXen72** [comment]
build failed because some other reason (daemon dissapeared), the check_formatting step passed

**Review by dessalines — APPROVED** [review]
I'll handle that one. Thx for this!

**Pull request merged**